misc/ls-extensions/modules/bal-shell-service/src/main/java/io/ballerina/shell/service/ConsoleOutCollector.java

```diff
@@ -50,6 +51,28 @@ public void write(int characterInt) throws IOException {
      * @return list of collected strings.
      */
     public List<String> getLines() {
+        // collecting if there is any left in the buffer
+        String buffered = collectFromBuffer();
+        if (!buffered.isEmpty()) {
+            lines.add(buffered);
+        }
         return lines;
     }
+
+    /**
+     * Collects bytes buffered in the buffer and clears the buffer.
+     *
+     * @return buffered string
+     */
+    private String collectFromBuffer() {
+        Byte[] byteObjects = buffer.toArray(new Byte[0]);
+        byte[] bytes = new byte[buffer.size()];
+        int idx = 0;
+        for (Byte b: byteObjects) {
```

Comment:
Shall we use `byte` here instead of `b` ?

Reply:
Updated in [89603b5](https://github.com/ballerina-platform/ballerina-lang/pull/36655/commits/89603b51365fafa24f6f44d2896fda1ed4f1982e) with `byt`